internal/cmd/postgresflex/user/delete/delete.go

```diff
@@ -45,7 +44,7 @@ func NewCmd() *cobra.Command {
 				`Delete a PostgreSQL Flex user with ID "xxx" for instance with ID "yyy"`,
 				"$ stackit postgresflex user delete xxx --instance-id yyy"),
 		),
-		Args: args.SingleArg(userIdArg, utils.ValidateUUID),
+		Args: args.SingleArg(userIdArg, nil),
```

Comment:
Is this only for the PotsgresFlex? What is the form of a user ID then?

Reply:
Seems to be 5 digit numbers